packages/react-next/src/components/Link/Link.types.ts

```diff
@@ -94,3 +92,19 @@ export interface ILinkStyleProps {
 export interface ILinkStyles {
   root: IStyle;
 }
+
+/**
+ * {@docCategory Link}
+ */
+export interface LinkSlots {
+  keytipData: React.ElementType;
```

Comment:
not very sure about making this a slot, do you see a use case of this?
anything we add to the public interface, it's going to be hard to remove.

Reply:
This is a weird one for me, I'd actually like to propose not bundling `Keytips` with the other components and have people wrap the components if they want this functionality. If we are indeed bundling them with the components then I don't see a reason why not to have them as slots. What do you think @xugao @dzearing @joschect?

Reply:
Moved it out of slots for now.